__tests__/tags.test.ts

```diff
@@ -74,7 +74,13 @@ describe('tags', () => {
     });
 
     it('should fetch all available tags when pagination is required', async () => {
-      const mockTagData = { data: [{ name: 'v2.0.0' }, { name: 'v2.0.1' }, { name: 'v2.0.2' }] };
+      const mockTagData = {
+        data: [
+          { name: 'v2.0.0', commit: { sha: 'abc123' } },
+          { name: 'v2.0.1', commit: { sha: 'def456' } },
+          { name: 'v2.0.2', commit: { sha: 'ghi789' } },
+        ],
+      };
       const expectedTags = mockTagData.data.map((tag) => tag.name);
```

Comment:
Can we change expectedTags here to also contain the `commitSha` from commit.sha.

Then we can remove the expectedDebugTags below and just mock to expectedTags